src/client/app/containers/ChartLinkContainer.ts

```diff
@@ -7,19 +7,25 @@ import ChartLinkComponent from '../components/ChartLinkComponent';
 import { State } from '../types/redux/state';
 
 function mapStateToProps(state: State) {
+	const chartType = state.graph.chartToRender;
 	let linkText = `${window.location.href}graph?`;
+	// let weeklyLink = ''; // reflects graph 7 days from present, with user selected meters and groups;
```

Comment:
Should there be a comment that this is a planned, future feature so leaving thie commented out code?